packages/nimble-components/src/patterns/required-visible/template.ts

```diff
@@ -0,0 +1,20 @@
+import { ViewTemplate, html, when } from '@microsoft/fast-element';
+import { iconAsteriskTag } from '../../icons/asterisk';
+import type { RequiredVisiblePattern } from './types';
+
+/* eslint-disable @typescript-eslint/indent */
+export function createRequiredVisibleLabelTemplate(
```

Comment:
We haven't done a good job of this in other patterns, but I think it'd be helpful to have a brief comment explaining what the pattern is for. Something like "Given the template for a control label, adds an icon next to the label to indicate that the input is required."